Realm/RLMApp.mm

```diff
@@ -223,7 +223,7 @@ static void setOptionalString(std::optional<std::string>& dst, NSString *src) {
 }
 
 - (NSString *)baseURL {
-    return getOptionalString(_config.base_url);
+    return getOptionalString(_config.base_url) ?: RLMStringViewToNSString(app::App::default_base_url());
```

Comment:
Shouldn't this be done by Core, and return the default url if there is no given url

Reply:
I guess it could, but they haven't done it 😄 I think it's a fair decision on their part as they probably want to keep the struct fairly simple and have the defaults be managed elsewhere.